src/spyglass/decoding/v1/clusterless.py

```diff
@@ -132,6 +132,9 @@ def make(self, key):
                     position_info.index <= interval_end,
                 )
             ] = True
+        is_training[
+            position_info[position_variable_names].isna().values.max(axis=1)
```

Comment:
What's the purpose of this line?

Reply:
To avoid using timepoints with `nan` values in any of the position variables in the encoding model